sklearn/covariance/_graph_lasso.py

```diff
@@ -303,16 +207,139 @@ def graphical_lasso(
         e.args = (e.args[0] + ". The system is too ill-conditioned for this solver",)
         raise e
 
+    return covariance_, precision_, costs, i + 1
+
+
+def graphical_lasso(
+    emp_cov,
+    alpha,
+    *,
+    cov_init=None,
+    mode="cd",
+    tol=1e-4,
+    enet_tol=1e-4,
+    max_iter=100,
+    verbose=False,
+    return_costs=False,
+    eps=np.finfo(np.float64).eps,
+    return_n_iter=False,
+):
+    """L1-penalized covariance estimator.
+
+    Read more in the :ref:`User Guide <sparse_inverse_covariance>`.
+
+    .. versionchanged:: v0.20
+        graph_lasso has been renamed to graphical_lasso
+
+    Parameters
+    ----------
+    emp_cov : ndarray of shape (n_features, n_features)
+        Empirical covariance from which to compute the covariance estimate.
+
+    alpha : float
+        The regularization parameter: the higher alpha, the more
+        regularization, the sparser the inverse covariance.
+        Range is (0, inf].
+
+    cov_init : array of shape (n_features, n_features), default=None
+        The initial guess for the covariance. If None, then the empirical
+        covariance is used.
+
+        .. deprecated:: 1.3
```

Comment:
Since the public class will call the private function `_graphical_lasso` then we will never need this `cov_init`. Indeed, looking at the code, it does nothing when calling `GraphicalLasso` because `cov_init` would always be equivalent to the empirical covariance.